EventListener/ViewResponseListener.php

```diff
@@ -75,16 +76,28 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             if ($configuration->getStatusCode() && (null === $view->getStatusCode() || Codes::HTTP_OK === $view->getStatusCode())) {
                 $view->setStatusCode($configuration->getStatusCode());
             }
+
+            // BC < 1.8
+            $viewClass = 'FOS\RestBundle\View\View';
+            if (get_class($view) == $viewClass) {
```

Comment:
You're right, but then we don't need this special case, do we? This should all be handled the same way if we only had the code from the `else` block, shouldn't it?